### ISSUE TITLE
Edit README.rst in examples/tutorial

### ISSUE THREAD

**Issue by spv7016** [pallets/flask#4950]
When trying to run the following command from README.rst:

$ flask --app flaskr run --debug

Receive the following message:
Usage: flask run [OPTIONS]
Try 'flask run --help' for help.
Error: No such option: --debug (Possible options: --debugger, --no-debugger)

It is needed to correct the command to:

$ flask --app flaskr run

To recieve the following (correct) message:
* Serving Flask app 'flaskr'
 * Debug mode: off
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
Running on http://127.0.0.1:5000
Press CTRL+C to quit

Environment:

- Python version: 3.11.1
- Flask version: 2.2.2

**Comment by davidism** [comment]
Looks fine to me: https://github.com/pallets/flask/tree/2.2.x/examples/tutorial#run Perhaps you're looking at main, you need to look at the git branch that matches the version you have installed.

**Comment by spv7016** [comment]
Indeed, you are correct (I haven't checked the relevant branch 2.2.x). 

It has been updated with the changed key word sequence to:
$ flask --app flaskr --debug run
(instead of previously stated: 
$ flask --app flaskr run --debug)

I've just checked, it returns correct (expected) output:
* Serving Flask app 'flaskr'
 * Debug mode: on
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on http://127.0.0.1:5000
Press CTRL+C to quit
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: ***-***-***
